modules/lib/src/main/resources/assets/js/v6/features/store/sidebarWidgets.store.ts

```diff
@@ -17,6 +17,8 @@ export const $sidebarWidgets = map<WidgetsStore>({
 });
 
 export function setActiveWidget(widget: Readonly<Widget> | undefined): void {
+    const existsInStore = $sidebarWidgets.get().widgets.some((p) => getWidgetKey(p) === getWidgetKey(widget));
+    if (!existsInStore) throw new Error('Widget not found in store');
```

Comment:
It's not really correct to throw an error here, because you are not handling it in the code later. And it's not something to be handled here. Just return.